package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@solarity/hardhat-migrate",
-  "version": "2.0.1",
+  "version": "2.1.1",
```

Comment:
This is not how versioning works

Reply:
Do breaking change anything?

Reply:
Yep, a few core dependencies has been updated